chainer/functions/connection/deconvolution_2d.py

```diff
@@ -0,0 +1,300 @@
+import math
+
+import numpy
+from six import moves
+
+from chainer import cuda
+from chainer import function
+from chainer.utils import conv
+from chainer.utils import type_check
+import ctypes
+
+if cuda.cudnn_enabled:
+    cudnn = cuda.cudnn
+    libcudnn = cuda.cudnn.cudnn
+    _fwd_pref = libcudnn.CUDNN_CONVOLUTION_FWD_SPECIFY_WORKSPACE_LIMIT
+
+
+def _pair(x):
+    if hasattr(x, '__getitem__'):
+        return x
+    return (x, x)
+
+
+def get_deconv_outsize(h, kh, sy, ph):
```

Comment:
I think `get_deconv_outsize` should be in `conv` module in parallel with `get_conv_outsize` .